datareservoirio/client.py

```diff
@@ -411,12 +418,26 @@ def get(
         else:
             df = pd.DataFrame(columns=("index", "values")).astype({"index": "int64"})
 
-        series = df.set_index("index").squeeze("columns").loc[start:end].copy(deep=True)
+        try:
+            series = (
+                df.set_index("index").squeeze("columns").loc[start:end].copy(deep=True)
+            )
+        except KeyError as e:
+            logging.warning("KeyError. The data will be sorted to attempt to resolve the issue. Please note that this operation may take some time.")
+            series = (
+                df.set_index("index")
+                .sort_index()
+                .squeeze("columns")
+                .loc[start:end]
+                .copy(deep=True)
+            )
         series.index.name = None
 
         if series.empty and raise_empty:  # may become empty after slicing
             raise ValueError("can't find data in the given interval")
```

Comment:
Here you might get the warning that you've gotten  a keyError and that we try to sort the data and then you will get another valueError saying that the program can't find data. Should i maybe copy the following code into each part of the try/except? 

series.index.name = None
if series.empty and raise_empty:  # may become empty after slicing
          raise ValueError("can't find data in the given interval")